write_test.go

```diff
@@ -0,0 +1,132 @@
+package dicom
+
+import (
+	"testing"
+	"os"
+	"encoding/binary"
+
+  "github.com/stretchr/testify/assert"
```

Comment:
it would be great to not depend this if we can avoid it. e.g. instead of 
```go
assert.Nil()
```

we can instead do stuff like 
```
if err != nil {
    t.Errorf("....")
}
```

At google we don't rely on assertion libraries so I kinda like that style now :). But we can revisit this in a follow up CL.